packages/core/src/Drawer/Drawer.styled.tsx

```diff
@@ -67,3 +68,40 @@ export const HeaderButton = styled(IconButton)`
     color: ${getPaletteColor('text.dark')};
   }
 `
+
+export const Handle = styled.div`
+  /* Handle */
+
+  /* Auto layout */
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  gap: 8px;
+
+  width: 48px;
+  height: 8px;
+
+  /* Background/Tint */
+  background: #d5dce4;
+  box-shadow: inset 0px 2px 2px rgba(0, 0, 0, 0.08);
+  border-radius: 999px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 0;
```

Comment:
Could you not inline the `DrawerHandle` component in the stories file if it is only used there?